dashboard/app/assets/javascripts/soundEffects.js

```diff
@@ -227,24 +227,35 @@ Sound.prototype.newPlayableBufferSource = function(buffer, options) {
   var newSound = this.audioContext.createBufferSource();
 
   // Older versions of chrome call this createGainNode instead of createGain
-  var gainNode;
   if (this.audioContext.createGain) {
-    gainNode = this.audioContext.createGain();
+    this.gainNode = this.audioContext.createGain();
   } else if (this.audioContext.createGainNode) {
-    gainNode = this.audioContext.createGainNode();
+    this.gainNode = this.audioContext.createGainNode();
   } else {
     return null;
   }
 
   newSound.buffer = buffer;
   newSound.loop = !!options.loop;
-  newSound.connect(gainNode);
-  gainNode.connect(this.audioContext.destination);
-  gainNode.gain.value = typeof options.volume === "undefined" ? 1 : options.volume;
+  newSound.connect(this.gainNode);
+  this.gainNode.connect(this.audioContext.destination);
+  var startingVolume = typeof options.volume === "undefined" ? 1 : options.volume;
+  this.gainNode.gain.setValueAtTime(startingVolume, this.audioContext.currentTime);
```

Comment:
Totally up to you. I just know that audio testing is a complex compatibility matrix. Perhaps leave it this way (because it make more sense) and if you see any issues during compat testing, switch it so the new code only executes for apps that exercise the "fade" code path?